hawkular-command-gateway-itest/src/test/java/org/hawkular/cmdgw/ws/test/EchoCommandITest.java

```diff
@@ -16,25 +16,125 @@
  */
 package org.hawkular.cmdgw.ws.test;
 
+import org.hamcrest.CoreMatchers;
+import org.hawkular.cmdgw.api.EchoRequest;
+import org.hawkular.cmdgw.api.WelcomeResponse;
+import org.hawkular.cmdgw.ws.test.TestWebSocketClient.ExpectedEvent;
+import org.hawkular.cmdgw.ws.test.TestWebSocketClient.ExpectedEvent.ExpectedClose;
+import org.hawkular.cmdgw.ws.test.TestWebSocketClient.ExpectedEvent.ExpectedFailure;
+import org.hawkular.cmdgw.ws.test.TestWebSocketClient.ExpectedEvent.ExpectedMessage;
+import org.hawkular.cmdgw.ws.test.TestWebSocketClient.MessageAnswer;
+import org.hawkular.cmdgw.ws.test.TestWebSocketClient.PatternMatcher;
 import org.testng.annotations.Test;
 
+import com.squareup.okhttp.Credentials;
+import com.squareup.okhttp.ws.WebSocket;
+
 /**
  * @author <a href="https://github.com/ppalaga">Peter Palaga</a>
  */
 
 public class EchoCommandITest extends AbstractCommandITest {
+    public static final long serverReauthPeriodSeconds =
+            Integer.parseInt(System.getProperty("org.hawkular.cmdgw.reauthIntervalSeconds"));
```

Comment:
Yes, it would fail and I wanted it like that. It is important to know which interval the server has set so that we can safely assume in the `testReAuthValidUser()` that the re-auth in AuthenticationService has happened at least once already.